plugins/woocommerce/tests/e2e/README.md

```diff
@@ -356,3 +356,58 @@ In the example above, you can see that `allows customer to see downloads` part o
 The test sequencer (`npx wc-e2e test:e2e`) includes support for saving [screenshots on test errors](https://github.com/woocommerce/woocommerce/tree/trunk/tests/e2e/env#test-screenshots) which can be sent to a Slack channel via a [Slackbot](https://github.com/woocommerce/woocommerce/tree/trunk/tests/e2e/env#slackbot-setup).
 
 For Puppeteer debugging, follow [Google's documentation](https://developers.google.com/web/tools/puppeteer/debugging).   
+
+## Running E2E Tests in the Windows Subsystem for Linux (WSL)
+
+The best way to setup your local development environment if you're on Windows is through the [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/). The following steps are for the Ubuntu 20.04 Linux distribution.
```

Comment:
```
The best way to setup your local development environment if you're on Windows is through the
```

@rodelgc The text looks good with just one suggestion to simplify this to

```
You can set up a local development environment on Windows with
```

The other comment I have is that the readme already has a section titled [Running tests](https://github.com/woocommerce/woocommerce/tree/add/wsl-instructions/plugins/woocommerce/tests/e2e#running-tests) and this content should be either in or referenced in that section. Having it at the end of the document makes the document disorganized. It's not a large amount of content but it seems like it would be too much of a disruption to put it all directly under `Running tests`. The alternative would be to make it a separate .md file and add a reference note at the beginning of the section. Do either of those sound good or do you have another idea for better organization.

@Konamiman Could you take a few minutes to review the instructions being added here to confirm they work for you?

Reply:
Thanks for the review, @rrennick! 

> You can set up a local development environment on Windows with

Done replacing the previous text with this one.

> The alternative would be to make it a separate .md file and add a reference note at the beginning of the section

I went with this approach. I created a `WSL_SETUP_INSTRUCTIONS.md` file that is referenced within the [Running tests](https://github.com/woocommerce/woocommerce/tree/add/wsl-instructions/plugins/woocommerce/tests/e2e#running-tests) section this way:
<img width="590" alt="Screen Shot 2021-11-05 at 8 29 11 PM" src="https://user-images.githubusercontent.com/4509348/140510420-fc51e5cc-e326-4dfb-be99-a7f7d9b11121.png">

And that this "WSL" section will just point to the `WSL_SETUP_INSTRUCTIONS.md` file like so:
<img width="577" alt="Screen Shot 2021-11-05 at 8 29 41 PM" src="https://user-images.githubusercontent.com/4509348/140510491-8d676431-c006-48eb-a35f-7972994e1b1a.png">

This way, readers will become aware right away that we do have WSL instructions just by reading our README file. Also, I think having a separate .md for WSL instructions will make it easy to expand it to other Linux distributions in the future.

Ready for another round of review!

Reply:
Thanks for the review, @rrennick! 

> You can set up a local development environment on Windows with

Done replacing the previous text with this one.

> The alternative would be to make it a separate .md file and add a reference note at the beginning of the section

I went with this approach. I created a `WSL_SETUP_INSTRUCTIONS.md` file that is referenced within the [Running tests](https://github.com/woocommerce/woocommerce/tree/add/wsl-instructions/plugins/woocommerce/tests/e2e#running-tests) section this way:

<img width="984" alt="Screen Shot 2021-11-05 at 8 47 05 PM" src="https://user-images.githubusercontent.com/4509348/140512385-00f29ff9-c584-483c-b67a-1627e8d1a821.png">


I think having a separate .md for WSL instructions will make it easy to expand it to other Linux distributions in the future.

Ready for another round of review!